app/views/projects/samples/index.html.erb

```diff
@@ -24,7 +41,23 @@
             "
           >
             <% @samples.each do |sample| %>
-            <tr class="text-sm font-normal text-slate-500 dark:text-slate-400">
+              <tr
+                id="<%= sample.id %>"
```

Comment:
**nitpick:** this would be better as `dom_id(sample)`. See https://api.rubyonrails.org/classes/ActionView/RecordIdentifier.html.

Does it need to be an id?  I usually only like to use id if it is something finicky that needs targeting by JavaScript, but with Stimulus we can use a target.  What are these id's for?

Reply:
I was trying to remove the successfully transferred samples from the table in a partial upload. I was trying to do it with Hotwire, like we do for personal tokens. In create.turbo_stream.erb, I do the following:
```html
<% sample_ids.each do |sample_id| %>
  <%= turbo_stream.remove sample_id %>
<% end %>
```
In order to use dom_id, I need to pass it the actual record. I only return a list of sample ids in the transfer service. I thought returning a list of samples was a bit much. Should I consider returning a list of objects in the transfer service instead?

Reply:
If I am reading this correctly, you probably just want to trigger a reload of the samples table.  The personal token table isn't paginated.

Reply:
Neither is this table, but it should be... working on adding pagination.

Reply:
This seems to work really well, I will let @deepsidhu85 ensure the backend code stuff is set up right.  My only nitpick here for the UI is that the bottom of the table looks different then the members table:

Samples:

![](https://github.com/phac-nml/irida-next/assets/11295750/c9a48e83-223d-4898-8efc-91a21ef6a6da)

Members:

![](https://github.com/phac-nml/irida-next/assets/11295750/a51a5a52-c81f-410d-ae79-98ffc6fd09f6)

Reply:
@joshsadam @ksierks Very easy fix to get rid of that bottom border showing under the pagination. In `app/views/projects/samples/index.html.erb` move `<%= turbo_frame_tag "project_samples_pagination" %>` from line 34 to line 37. It should be above the last 2 `div`s.

Reply:
Thanks. Please see 838a56fcd0bdaeb2b90d97014b9565f7b140146f.